stable/kommander/values.yaml

```diff
@@ -42,3 +42,40 @@ ingress:
   # extraAnnotations:
   #   ingress.kubernetes.io/foo: bar
   path: /ops/portal/kommander
+
+federate:
+  addons: true
+  roles: true
+  rolebindings: true
+
+# This name must match the name of the kubefed dep
+# in order to override values.
+kubefed:
+  controllermanager:
+    enabled: true
+    # override value is necessary due to a "featureGates":interface {}(nil)... spec.featureGates in body must be of type array: "null"
+    # bug found
+    featureGates:
+      CrossClusterServiceDiscovery: "Enabled"
+      FederatedIngress: "Enabled"
+      PushReconciler: "Enabled"
```

Comment:
I wonder if, for now, we just enable `PushReconciler` as that's all we're actually using?

Reply:
i agree.

Reply:
sure, but not sure the benefit/drawbacks of this.